.bazelrc

```diff
@@ -12,276 +12,10 @@
-###############################################################################
-
-build:windows --define=iree_is_msvc=true
-build:windows --copt=/wd4624 # destructor was implicitly defined as deleted
-build:windows --copt=/wd4244 # possible loss of data
-build:windows --copt=/wd4005 # macro redefinition
-build:windows --copt=/wd4267 # initializing: possible loss of data
-build:windows --copt=/wd4141 # inline used more than once
-build:windows --per_file_copt=mkl_dnn@/wd4551 # missing argument list
-build:windows --per_file_copt=mkl_dnn@/wd4068 # unknown pragma
-build:windows --per_file_copt=farmhash@/wd4319 # zero extending to T of greater size
-
-build:windows --linkopt=/IGNORE:4217 # mismatch import/export declspec
-
-# Enables unix-style runfiles link trees (requires symlink permission).
-# See: https://blogs.windows.com/windowsdeveloper/2016/12/02/symlinks-windows-10/
-# Generally: Enable Developer Mode in the Developer Settings page of the
-# system settings.
-build:windows --experimental_enable_runfiles
-# Flags to make tensorflow build.
-# Some of these are also of general use and fine to enable globally for windows.
-build:windows --copt=/arch:AVX
-# Host and target are the same in windows so don't waste time building both.
-build:windows --distinct_host_configuration=false
-# Avoids incompatible versions of winsock and other badness.
-build:windows --copt=/DWIN32_LEAN_AND_MEAN --host_copt=/DWIN32_LEAN_AND_MEAN
-# That is one way to have less warnings :(
-build:windows --per_file_copt=tensorflow@-w
-build:windows --per_file_copt=protobuf@-w
-# Why are min/max macros? No one knows.
-build:windows --copt=/DNOMINMAX --host_copt=/DNOMINMAX
-# Yay for security warnings. Boo for non-standard.
-build:windows --copt=/D_CRT_SECURE_NO_WARNINGS --host_copt=/D_CRT_SECURE_NO_WARNINGS
-# TensorFlow requires the "monolithic" build mode for now on Windows.
-build:windows --define framework_shared_object=false
-# Necessary for M_* math constants.
-build:windows --copt=/D_USE_MATH_DEFINES --host_copt=/D_USE_MATH_DEFINES
-
-# Workaround WinGDI.h defining `ERROR`, which conflicts with logging macros.
-# Note that IREE and TensorFlow both `#undef ERROR` and define their own
-# separate logging constants with the same name, but IREE needs the Windows
-# "graphics device interface" (GDI) for certain GUI sample projects.
-build:windows --per_file_copt=tensorflow@-DNOGDI
-
-# Disables TensorFlow eigen bloat and reduces compile times.
-build:windows --define=override_eigen_strong_inline=true
-
-# Another TensorFlow flag from their config script.
-build:windows --define with_default_optimizations=true
-
-###############################################################################
+# Import the main bazelrc config. This is in a separate file so that it's easy
+# to turn off all user bazelrc options by specifying
+# `--nosystem_rc --nohome_rc --noworkspace_rc --bazelrc=build_tools/bazel/iree.bazelrc`
+try-import %workspace%/build_tools/bazel/iree.bazelrc
```

Comment:
I'm on the fence about this being nested under `./build_tools/bazel/`.  Each of the other `*.bazelrc` files is in the root... but we'd like to minimize the number of files in the root if possible. I guess the parallel to https://github.com/google/iree/blob/main/build_tools/cmake/iree_copts.cmake makes this less of an issue?

Reply:
The other bazelrc are added by users though and in the `.gitignore`. They seem much more akin to user dot files and as untracked files need to be easy to find. My rationale here was that we should stick as much build_tools related stuff under build_tools. Honestly I thought you'd be thrilled that I didn't add another file at root :stuck_out_tongue: